### PR TITLE
[Bugfix:CourseMaterials] Disables scroll on manage grader/student

### PR DESCRIPTION
**Pull request by RayReyRam**
### Please check if the PR fulfills these requirements:

* [ ] Tests for the changes have been added/updated (if possible)
* [ ] Documentation has been updated/added if relevant
* [ ] Screenshots are attached to Github PR if visual/UI changes were made

### What is the current behavior?

Previously there was an ability, when a popup was open to scroll in the background behind/around the modal. 
related to #10200

### What is the new behavior?

On the instructor side in the sections manage grader and manage students when clicking on a popup the scrolling ability is disabled and reenabled when out of the popup 

### Other information?
To test enter as instructor, on side bar look for manage grader and or manage student open the popups there

### CODE REVIEW

**Review by JoJo-ESC — APPROVED** [review]
Functionality checked. All possible pop up opportunities checked.

**Review by JoJo-ESC — COMMENTED** [review]
Functionality checked. All popups checked in the process of review. fIx the test issue and ill approve.

File: site/public/js/directory.js [hunk 1]
```diff
@@ -5,7 +5,7 @@
 function newDownloadForm() {
     $('.popup-form').css('display', 'none');
     const form = $('#download-form');
-    form.css('display', 'block');
+    showPopup('#download-form');
```
Comment (jeffrey-asm) [inline site/public/js/directory.js:8]: For all the JS files referencing this method, just add the method to the global comment at the top of the file like /* global ... showPopup */ and these errors will go

**Review by jeffrey-asm — CHANGES_REQUESTED** [review]
Most of the changes look good, just fix the linting issues and there may be a potential issue where the "Upload Student Photos" still allows scrolling on the student photos page popup, but I'm not sure if this page was a part of the issue.

**Comment by RayReyRam** [comment]
> Most of the changes look good, just fix the linting issues and there may be a potential issue where the "Upload Student Photos" still allows scrolling on the student photos page popup, but I'm not sure if this page was a part of the issue.

Is this upload student photos button in the mange student section?

**Comment by jeffrey-asm** [comment]
> > Most of the changes look good, just fix the linting issues and there may be a potential issue where the "Upload Student Photos" still allows scrolling on the student photos page popup, but I'm not sure if this page was a part of the issue.
> 
> Is this upload student photos button in the mange student section?

No, but they are related, so it would make reasonable to address it.

**Review by ZheyuDeng — APPROVED** [review]
Tested on Chrome works for me. The issue in user profile should be addressed in different pr

**Pull request merged**